README.md

```diff
@@ -170,6 +170,9 @@ Typical value should be between 0.01 .. 999.99
 
 - **bool zeroCalibration()** to be called after at least 5 minutes in fresh air.
 See example sketch.
+- **bool manualZeroCalibration(uint16_t value)** for v118 sensors, set the zero calibration value manually.
```

Comment:
.h file says 
bool                manualZeroCalibration(uint16_t value = 0);
need to add default value to documentation. 

maybe add a 117 comment?

Reply:
Mirrored the comment I had in the source file.